packages/driver/cypress/integration/commands/actions/click_spec.js

```diff
@@ -1178,6 +1178,42 @@ describe('src/cy/commands/actions/click', () => {
         })
       })
 
+      it('does not scroll when scrollToElement is false in options', () => {
```

Comment:
Can we have a few tests to ensure the scroll actually doesn't happen when passed the option? Just in case something with the `scrollIntoView` function gets changed in the future, to catch that the actual scrolling is broken / not broken.

Reply:
added tests that assert it is not scrolled.